.rubocop.yml

```diff
@@ -34,7 +33,7 @@ Lint/UselessAccessModifier:
 Metrics/AbcSize:
   Max: 20
 Metrics/ClassLength:
-  Max: 240
+  Max: 300
```

Comment:
Here it is for the record: https://github.com/jekyll/jekyll/pull/4960#discussion_r65001711